title-updater.js

```
@@ -0,0 +1,103 @@
+        }
+    }
+
+    // Update title when page loads
+    if (document.readyState === 'loading') {
+        document.addEventListener('DOMContentLoaded', updatePageTitle);
+    } else {
+        updatePageTitle();
+    }
+
+    // Also update when navigating (for SPAs)
+    if (window.history && window.history.pushState) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)